go.mod

```diff
@@ -4,40 +4,133 @@ go 1.17
 
 require (
 	cloud.google.com/go v0.81.0
-	github.com/evanphx/json-patch v4.11.0+incompatible
+	github.com/evanphx/json-patch v4.12.0+incompatible
 	github.com/gofrs/flock v0.7.1
 	github.com/google/go-cmp v0.5.6
 	github.com/google/go-tpm v0.3.2
-	github.com/google/uuid v1.1.4 // indirect
-	github.com/imdario/mergo v0.3.11 // indirect
 	github.com/prometheus/client_golang v1.11.0
 	github.com/spf13/cobra v1.2.1
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.7.0
-	golang.org/x/oauth2 v0.0.0-20210427180440-81ed05c6b58c
+	golang.org/x/oauth2 v0.0.0-20210819190943-2bc19b11175f
 	google.golang.org/api v0.46.0
-	gopkg.in/check.v1 v1.0.0-20200902074654-038fdea0a05b // indirect
 	gopkg.in/gcfg.v1 v1.2.0
 	gopkg.in/warnings.v0 v0.1.2
-	k8s.io/api v0.22.2
-	k8s.io/apimachinery v0.22.2
-	k8s.io/apiserver v0.22.0
+	k8s.io/api v0.23.1
+	k8s.io/apimachinery v0.23.1
+	k8s.io/apiserver v0.23.1
 	k8s.io/client-go v9.0.0+incompatible
-	k8s.io/cloud-provider v0.22.0
+	k8s.io/cloud-provider v0.23.1
 	k8s.io/cloud-provider-gcp/providers v0.0.0
-	k8s.io/code-generator v0.22.2
-	k8s.io/component-base v0.22.0
-	k8s.io/component-helpers v0.22.0
-	k8s.io/controller-manager v0.22.0
-	k8s.io/klog/v2 v2.9.0
-	k8s.io/kube-controller-manager v0.22.0
-	k8s.io/kubelet v0.22.0
-	k8s.io/kubernetes v1.22.0
-	k8s.io/metrics v0.22.0
-	k8s.io/utils v0.0.0-20210707171843-4b05e18ac7d9
+	k8s.io/code-generator v0.23.1
+	k8s.io/component-base v0.23.1
+	k8s.io/component-helpers v0.23.1
+	k8s.io/controller-manager v0.23.1
+	k8s.io/klog/v2 v2.30.0
+	k8s.io/kube-controller-manager v0.23.1
+	k8s.io/kubelet v0.23.1
+	k8s.io/kubernetes v1.23.0
```

Comment:
Is there a reason we keep kubernetes at 1.23.0 and others at 23.1?

Reply:
This is a mistake. It should be `1.23.1`. However, one can argue that we should eliminate the dependency of k/k soon.

Reply:
```suggestion
	k8s.io/kubernetes v1.23.1
```

Reply:
Nice catch! I updated the version.